images/primary_node/repo_url_insert_script.sh

```diff
@@ -17,17 +17,8 @@
 MAPR_VERSION="$1"
 MEP_VERSION="$2"
 
-if [ "$MAPR_VERSION" = "" ]; then
-    echo "Usage: $0 MAPR_VERSION ..."
-    exit 1
-fi 
-if [[ "$MAPR_VERSION" != "5.2.0" && "$MEP_VERSION" == "" ]]; then
-    echo "Usage: $0 MAPR_VERSION MEP_VERSION ..."
-    exit 1
-fi
-
 echo "baseurl = http://package.mapr.com/releases/v${MAPR_VERSION}/redhat/" >> /etc/yum.repos.d/mapr_core.repo
-if [[ $MAPR_VERSION = "5.2.0" && $MEP_VERSION == "" ]]; then
+if [ $MAPR_VERSION = "5.2.0" ] && [ -z "${MEP_VERSION}" ]; then
```

Comment:
For consistency (and as a safeguard against missing args), use `"${MAPR_VERSION}"` (always use double quotes and `${ }` syntax)